mysql/datadog_checks/mysql/data/conf.yaml.example

```
@@ -86,37 +86,32 @@ instances:
     ## @param use_global_custom_queries - string - optional - default: true
     ## How `global_custom_queries` should be used for this instance. There are 3 options:
     ##
-    ## * 'true' - `global_custom_queries` override `custom_queries`.
-    ## * 'false' - `custom_queries` override `global_custom_queries`.
-    ## * 'extend' - `global_custom_queries` are used in addition to any `custom_queries`.
+    ## 1. true - `global_custom_queries` override `custom_queries`.
+    ## 2. false - `custom_queries` override `global_custom_queries`.
+    ## 2. extend - `global_custom_queries` are used in addition to any `custom_queries`.
```

```suggestion
    ## 3. extend - `global_custom_queries` are used in addition to any `custom_queries`.
```
or is this markdown where it doesn't matter?

---

This isn't markdown so it's def a typo